src/components/data_submitter/DataAccessGovernance.js

```diff
@@ -0,0 +1,168 @@
+      setDacs(dacList);
+    });
+  }, [])
+
+  const addNewConsentGroup = () => {
+    const newConsentGroups = cloneDeep(consentGroups);
+    newConsentGroups.push({});
+    setConsentGroups(newConsentGroups)
+  };
+
+  const deleteConsentGroup = (idx) => {
+    const newConsentGroups = cloneDeep(consentGroups);
+    newConsentGroups[idx] = undefined;
+    setConsentGroups(newConsentGroups);
+  };
+
+  const updateConsentGroup = (idx, update) => {
+    const newConsentGroups = cloneDeep(consentGroups);
+    newConsentGroups[idx] = {
+      ...consentGroups[idx],
+      ...update,
+    };
+    updateFormData({
+      consentGroups: newConsentGroups,
+    });
+  };
+
+
+
+  return div({}, [
+    div({}, [
+
+      // open or closed access
+      div({
+        style: radioContainer,
+      }, [
+        RadioButton({
+          style: radioButtonStyle,
+          id: 'alternativeDataSharingPlanControlledOpenAccess',
+          name: 'alternativeDataSharingPlanControlledOpenAccess',
+          value: 'alternativeDataSharingPlanControlledOpenAccess',
+          defaultChecked: formData.alternativeDataSharingPlanControlledOpenAccess === OPEN_ACCESS,
+          onClick: () => updateFormData({
+            alternativeDataSharingPlanControlledOpenAccess: OPEN_ACCESS,
+          }),
+          description: 'Open Access',
+        }),
+      ]),
+
+      div({
+        style: radioContainer,
+      }, [
+        RadioButton({
+          style: radioButtonStyle,
+          id: 'alternativeDataSharingPlanControlledOpenAccess',
+          name: 'alternativeDataSharingPlanControlledOpenAccess',
+          value: 'alternativeDataSharingPlanControlledOpenAccess',
+          defaultChecked: formData.alternativeDataSharingPlanControlledOpenAccess === CLOSED_ACCESS,
+          onClick: () => updateFormData({
+            alternativeDataSharingPlanControlledOpenAccess: CLOSED_ACCESS,
+          }),
+          description: 'Closed Access',
+        }),
+      ]),
+
+      
+      div({
+        isRendered: formData.alternativeDataSharingPlanControlledOpenAccess === CLOSED_ACCESS,
+      },
+        [
+          // select dac
+
+          h(SearchSelect, {
+            id: 'data_submission_select_dac',
+            name: 'data_submission_select_dac',
+            onSelection: (selection) => updateFormData({
+              dataAccessCommitteeId: selection,
+            }),
+            options: dacs.map((dac) => {
+              return { key: dac.dacId, displayText: dac.name };
+            }),
+            placeholder: 'Search for institution...',
+            value: formData.dataAccessCommitteeId,
+          }),
+
+
+          // add consent groupa
+          div({className: 'right-header-section', style: {
+            display: 'flex',
+            alignItems: 'flex-end'
+          }}, [
+            a({
+              id: 'btn_addInstitution',
+              className: 'btn-primary btn-add common-background',
+              onClick: () => addNewConsentGroup(),
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 12 spaces but found 14.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9969451)
```suggestion
            onClick: () => addNewConsentGroup(),
```